riscv/core.py

```diff
@@ -64,12 +64,13 @@ def vscale_core(clock,
     dmem_rdata = Signal(intbv(0)[HASTI_BUS_WIDTH:0])
     dmem_badmem_e = Signal(intbv(0)[1:])
 
-    @always_comb
+    @instance
     def assign():
         htif_ipi_req_valid.next = intbv(0)[1:]
         htif_ipi_req_data.next = intbv(0)[1:]
         htif_ipi_resp_ready.next = intbv(1)[1:]
         htif_debug_stats_pcr.next = intbv(0)[1:]
+        yield delay(1)
```

Comment:
Actually from the logic, there is no need for it, just added it as a safe-side measure. Will remove it.